run

```diff
@@ -4,6 +4,4 @@
 #
 # Note also that symlinks do not work in the RT_SiteConfig.d directory.
 cp /opt/share/secrets/rt/rt-db /etc/request-tracker4/RT_SiteConfig.d/98-ocfdb.pm
-cp /opt/share/secrets/rt/rt.keytab /tmp/rt.keytab
-chown www-data:root /tmp/rt.keytab
```

Comment:
what was this used for and why is it no longer necessary?

Reply:
This was for when we used mod_auth_kerb for authentication. It should have been removed when switching to PAM.